fiat-core/src/main/java/com/netflix/spinnaker/fiat/model/Authorization.java

```diff
@@ -18,14 +18,14 @@
 
 import java.util.Arrays;
 import java.util.Collections;
-import java.util.HashSet;
+import java.util.LinkedHashSet;
 import java.util.Set;
 
 public enum Authorization {
   READ,
   WRITE,
   EXECUTE;
 
-  public static Set<Authorization> ALL =
-      Collections.unmodifiableSet(new HashSet<>(Arrays.asList(values())));
+  public static final Set<Authorization> ALL =
+      Collections.unmodifiableSet(new LinkedHashSet<>(Arrays.asList(values())));
```

Comment:
`Collections.unmodifiableSet(EnumSet.allOf(Authorization.class))`

(also uses natural iteration order, presuming that's why you switched to `LinkedHashSet`)